app/components/UI/Tokens/index.tsx

```diff
@@ -218,25 +232,61 @@ const Tokens: React.FC<TokensI> = ({ tokens }) => {
   const onActionSheetPress = (index: number) =>
     index === 0 ? removeToken() : null;
 
+  const isTokenFilterEnabled = process.env.PORTFOLIO_VIEW === '1';
```

Comment:
```suggestion
  const isTokenFilterEnabled = process.env.PORTFOLIO_VIEW === 'true';
```
Let's keep the flag values consistent